### DISCUSSION TITLE
grain boundry misorientation

### DISCUSSION THREAD

**Discussion by lyg1232** [mtex-toolbox/mtex#1517] [Ask Anything]
Hello everyone：
I have encountered a problem now, and I would like to ask you for help. In the EBSD data of a magnesium alloy, I selected some characteristic grains, as shown in the figure below. Now, I want to extract the grain boundry misorientation from these grains, how can I achieve it？
![cb32664dc813cfe2afe3fa27303d158](https://user-images.githubusercontent.com/81472625/182286458-a62dc7dc-049f-4b7a-a047-a7e67a050741.png)
thank you for your reply！！！！！

**Comment by kilir** [comment]
Hi,
have a look at the documentation.
https://mtex-toolbox.github.io/BoundaryProperties.html
Hope this helps,
Rüdiger
